app/src/main/res/layout/offer_footer.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:background="?android:colorBackground"
+    android:orientation="vertical"
+    android:paddingHorizontal="@dimen/base_margin_double"
+    android:paddingTop="@dimen/base_margin_sextuple"
+    android:paddingBottom="150dp">
+
+    <TextView
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:gravity="center"
+        android:text="@string/OFFER_FOOTER_SUBTITLE"
+        android:textAppearance="?textAppearanceSubtitle2" />
+
+    <Button
+        android:id="@+id/chat_button"
+        style="?materialButtonOutlinedLargeStyle"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/base_margin_double"
+        android:text="@string/OFFER_FOOTER_BUTTON_TEXT" />
+
+    <TextView
+        android:id="@+id/text"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/base_margin_sextuple"
+        android:clickable="true"
+        android:focusable="true"
+        android:linksClickable="true"
```

Comment:
Are all of the following attributes required in order to support in-view links? 🤔

Reply:
Yes :O